app/api/sessions.py

```diff
@@ -194,11 +197,32 @@ def before_get_object(self, view_kwargs):
             event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'identifier')
             view_kwargs['event_id'] = event.id
 
+    def after_update_object(self, session, data, view_kwargs):
+        """ Send email if session accepted or rejected """
+        if session.state == 'accepted' or session.state == 'rejected':
+            # Email for speaker
+            speakers = session.speakers
+            for speaker in speakers:
+                frontend_url = get_settings()['frontend_url']
+                link = "{}/{}/sessions/{}" \
```

Comment:
I think the link will give wrong value. It should be `"{}/events/{}/sessions/{}"`